go.mod

```diff
@@ -1,28 +1,28 @@
 module github.com/envoyproxy/ai-gateway
 
-go 1.23.2
+go 1.23.4
 
 replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.16
 
 require (
 	github.com/aws/aws-sdk-go-v2 v1.32.8
 	github.com/aws/aws-sdk-go-v2/aws/protocol/eventstream v1.6.7
 	github.com/aws/aws-sdk-go-v2/config v1.28.10
-	github.com/envoyproxy/gateway v1.2.4
+	github.com/envoyproxy/gateway v0.5.0-rc.1.0.20250115172926-8b89dadfbd2c
```

Comment:
will be fixed to EG v1.3.0 once it's released, and in the meantime we will use the latest for e2e